internal/handlers/telegram/helpers.go

```diff
@@ -5,12 +5,25 @@ import (
 )
 
 /*
-GetUsername returns the name and username of a user. Since usernames are optional
+ResolveUserName does basic cleanup if a user does not have a username on Telegram.
+*/
+func ResolveUserName(u *tgbotapi.User) string {
+	if u.UserName == "" {
+		return u.FirstName
+	}
+	// Add ZWSP to prevent pinging across platforms
+	// See https://github.com/42wim/matterbridge/issues/175
+	username := u.UserName[:1] + "" + u.UserName[1:]
+	return username
+}
+
+/*
+GetFullUsername returns the name and username of a user. Since usernames are optional
 on Telegram, we first need to check to see if they have one set.
 */
-func GetUsername(u *tgbotapi.User) string {
+func GetFullUsername(u *tgbotapi.User) string {
 	if u.UserName == "" {
 		return u.FirstName
 	}
-	return u.FirstName + " (@" + u.UserName + ")"
+	return u.FirstName + " (@" + u.UserName[:1] + "" + u.UserName[1:] + ")"
 }
```

Comment:
I am a little confused, what makes `ResolveUserName` and `GetFullUsername` different here?

Reply:
`ResolveUserName` is the simple version of `GetFullUsername`. It is used any time we need a person's username. 

`GetFullUsername` is *only* used in the `join/part` handlers, which gets both the person's first name, *and* their Telegram username. This is the same setup we used in the NodeJS version.

Reply:
@Tjzabel Makes sense, thanks for the answer. Should have read your code comments too, I didn't see them in the diff. I have two new pieces of feedback, will open a new thread for these.